### PR TITLE
bpo-23878: Remove an unneeded fseek() call in _Py_FindEnvConfigValue()

### PR DESCRIPTION
**Pull request by ZackerySpytz**
https://bugs.python.org/issue23878

### CODE REVIEW

File: Python/pathconfig.c [hunk 1]
```diff
@@ -741,7 +741,9 @@ _Py_FindEnvConfigValue(FILE *env_file, const wchar_t *key,
     char buffer[MAXPATHLEN * 2 + 1];  /* allow extra for key, '=', etc. */
     buffer[Py_ARRAY_LENGTH(buffer)-1] = '\0';
```
Comment (vstinner) [inline Python/pathconfig.c:743]: Please update the function comment: explain that it returns 1 and copy it into value if found, or return 0 on error or if not found. Something like that.

I'm not sure why key is a parameter: its value is always "home". Maybe write a following PR to remove the key parameter? (need to update getpath.c and getpathp.c)

Comment (vstinner) [reply Python/pathconfig.c:743]: Note: I'm not sure why fseek(0) is called: we just opened the file in read mode, not in append mode, the file cursor should already be 0.

Comment (ZackerySpytz) [reply Python/pathconfig.c:743]: Thank you very much for your review, @vstinner. I have removed the `fseek()` call -- it indeed seems to be unneeded.
I will create a pull request to remove the key parameter.

**Comment by vstinner** [comment]
I don't think that it's worth it to backport this change to Python 3.7 or 3.8.

**Pull request merged**